source/coap_security_handler.c

```diff
@@ -18,15 +18,24 @@
 #include "randLIB.h"
 #include "mbedtls/ssl_ciphersuites.h"
 
-
-const int PSK_SUITES[] = {
+#if defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)
+const int ECJPAKE_SUITES[] = {
     MBEDTLS_TLS_ECJPAKE_WITH_AES_128_CCM_8,
     0
 };
+#endif
+
+const static int PSK_SUITES[] = {
+    MBEDTLS_TLS_PSK_WITH_AES_128_CBC_SHA256,
+    MBEDTLS_TLS_PSK_WITH_AES_256_CCM_8,
+    MBEDTLS_TLS_PSK_WITH_AES_128_CCM_8,
+    0
```

Comment:
this flagging mandates that you either select PSK or ECJPAKE, but those need to live in same device and user of the service api selects which is used in certain service instance